include/sys/dirent.h

```diff
@@ -66,7 +66,7 @@ struct dirent {
     off_t     d_off;              /**< \brief File offset */
     uint16_t  d_reclen;           /**< \brief Record length */
     uint8_t   d_type;             /**< \brief File type */
-    char      d_name[];           /**< \brief Filename */
+    char      d_name[NAME_MAX];   /**< \brief Filename */
```

Comment:
```suggestion
    char      d_name[0];          /**< \brief Filename */
```

Reply:
Yeah I can confirm this little trick works with GCC 13.2.0!

Reply:
Awesome. I'm making a note to do this in the future when I need FAM behavior.... Sorry about breaking C++!